esm/package.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
If the library paths is mapped on `.` for ESM, is there a need to create the `/esm` directory?

The purpose we have these dirs in `msw` is to enable imports like `msw/node` where the structure is:

```
msw/
  node/
    package.json
```

But that's not the case with ESM in this package, is it? Or is it intended to be imported like `@mswjs/cookies/esm`? 🤔

Reply:
We could probably avoid it. I'll check locally when I get some time and remove if that works as well